### ISSUE TITLE
Add JSON output of bestiary.

### ISSUE THREAD

**Issue by cfi2017** [haswellr/CritterDB#14]
The application should provide a link to output a bestiary according to the format specified by 5etools:
- https://5etools.com/data/bestiary/bestiary-cos.json
- https://5etools.com/converter.html

This URL can be used to import a bestiary into Roll20 using 5etools import tool.

**Comment by DMsGuild201** [comment]
(I'd suggest using this schema basis)

https://github.com/TheGiddyLimit/TheGiddyLimit.github.io/blob/master/test/schema/bestiary/bestiary.json

**Comment by haswellr** [comment]
I've made a series of changes which let you export bestiaries and/or individual critters in the 5ETools format. However, it does not yet generate a URL. I'm unsure if I want to go through that effort, preferring instead that 5E Tools allows you to import text or a file from your computer straight to Roll20. This is more secure in terms of permissions, and doesn't force CritterDB to serve a new endpoint purely for 5E Tools to function.

Regardless, I've created a new issue to track that while I think on it. In the mean time, I'm going to resolve this one as the core work of making bestiaries exportable to 5E tools is complete. Follow-up issue: https://github.com/haswellr/CritterDB/issues/59
